drakrun/drakrun/draksetup.py

```diff
@@ -437,6 +481,11 @@ def postinstall(report, generate_usermode):
 
     install_info = InstallInfo.load()
 
+    logging.info("Cleaning up leftovers(if any)")
+
+    for _ in os.listdir(PROFILE_DIR):
+        safe_delete(os.path.join(PROFILE_DIR, _))
```

Comment:
`_` should be used only as a placeholder when it's not used
```suggestion
    for profile in os.listdir(PROFILE_DIR):
        safe_delete(os.path.join(PROFILE_DIR, profile))
```